openid-4-verifiable-presentations-1_0.md

```diff
@@ -310,6 +313,9 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`wallet_attestation`:
+: OPTIONAL. A boolean value that indicates whether or not a Wallet Attestation is requested by the Verifier. If the parameter is not present, the default value is `false`. If the value is `true`, a Wallet Attestation is expected to be part of the response (see `wallet_attestation` in (#response-parameters)).
```

Comment:
I'm fine with this approach if we can ensure that we leave the door open to later (in future versions) add a DCQL query or other complex query here if needed.

Not sure if this is the best way:

```suggestion
: OPTIONAL. A value that indicates whether or not a Wallet Attestation is requested by the Verifier. If the parameter is not present, or is `false`, no Wallet Attestation is requested. Otherwise, a Wallet Attestation is expected to be part of the response (see `wallet_attestation` in (#response-parameters)).
```

Reply:
Maybe add something like "Note: Instead of offering just the boolean values `true`/`false`, extensions of this specifications may define custom queries to be used in this parameter. Therefore, any value other than `false` MUST be treated as a request for a Wallet Attestation."